docs/csharp/language-reference/compiler-messages/toc.yml

```diff
@@ -3,218 +3,1550 @@
   items:
   - name: Compiler error CS0001
     href: cs0001.md
+  - name: Compiler Error CS0003
```

Comment:
```suggestion
  - name: Compiler error CS0003
```

Reply:
replaced in all locations (and warnings)